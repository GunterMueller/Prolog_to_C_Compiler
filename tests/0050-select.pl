main :-
	select(3, [1,2,3], [1,2]),
	select(X, Y, [1,2]), display(X), nl, fail.
main :-
	select(X, [1,2,3], Y), display(X), put(32), display(Y), nl, fail.
main.

