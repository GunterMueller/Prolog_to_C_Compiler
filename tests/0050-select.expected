_9
_9
_9
1 [2,3]
2 [1,3]
3 [1,2]
